setup.py

```diff
@@ -8,7 +8,8 @@ def setup_package():
 
     setup(
         name="streamlit_modal",
-        version="0.0.5",
+        install_requires=['streamlit', 'deprecation'],
+        version="0.0.6",
```

Comment:
Please change this to 0.1.0.

Reply:
done f75545600d023a331606a13833f6c197e9ded73d